compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -453,7 +453,9 @@ static bool isFusableWithConsumer(
   // all operations can bufferize without needing additional memory.
   for (OpOperand *inputOperand : consumerLinalgOp.getDpsInputOperands()) {
     if (inputOperand->get().getDefiningOp() != producer) continue;
-    if (isa<linalg::ConvolutionOpInterface>(producer) &&
+    if ((isa<linalg::ConvolutionOpInterface>(producer) ||
+         (!aggressiveFusion &&
```

Comment:
I dont want to guard this by `aggressiveFusion`... We either drop it or we try to get the backend to handle such dispatch. I am fine with dropping it... only affects quantized models from what I saw....

Question though is this happens to prevent the fusion, but is not the root cause here....

Reply:
fwiw int8 quantized models and their performance will be high on our hit list soon.